settings.gradle

```diff
@@ -0,0 +1,2 @@
+rootProject.name = 'wiremocksample'
```

Comment:
Can you change this to `ratpack-kotlin-demo`?

Reply:
Yep, Just changed that.